### PR TITLE
Added BFR API endpoints to new Platform function

### PR DESCRIPTION
**Pull request by WolfyUK**
### Context
[AB#214807](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/214807) [AB#188391](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/188391)

### Change proposed in this pull request
Created new Platform function to return BFR and BFR metrics. Moved prior Web stubs into their own API and updated all affected references (may cause merge conflicts with other pending PRs).

### Guidance to review 
Swagger may be use to test new endpoints but without any matching data in the respective database tables, an empty array will always be returned.

### Checklist
- [x] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)
- [x] Your code builds clean without any errors or warnings
- [ ] ~~You have run all unit/integration tests and they pass~~
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

File: platform/src/apis/Platform.Api.Insight/Balance/BalanceModel.cs
Comment (jrabbott) [inline platform/src/apis/Platform.Api.Insight/Balance/BalanceModel.cs:1]: `BudgetForecastReturnModel` mode shouldn't be in the `BalanceModel.cs`

Comment (WolfyUK) [reply platform/src/apis/Platform.Api.Insight/Balance/BalanceModel.cs:1]: Good spot!

Comment (jrabbott) [reply platform/src/apis/Platform.Api.Insight/Balance/BalanceModel.cs:1]: Still there?

Comment (WolfyUK) [reply platform/src/apis/Platform.Api.Insight/Balance/BalanceModel.cs:1]: Weird. No idea what happened in `b81e31`

**Review by jrabbott — DISMISSED** [review]

**Review by jrabbott — APPROVED** [review]

**Pull request merged**